discover/commerce/articles/01-catalog-management/01-product-categories.markdown

```diff
@@ -0,0 +1,112 @@
+# Product Categories [](id=product-categories)
+
+Products can be organized into *Categories*. These groupings allow multiple
+products to be handled as a single unit. For example, a product category could
+be used to apply discounts or other offers to a set of products, to assist
+buyers in finding products, or to promote certain products to a specified user
+segment.
+
+To organize products into categories, you must create the categories first and
+then assign products to them.
+
+## Creating Categories [](id=creating-categories)
+
+To create vocabularies, categories and subcategories, go to *Site Menu* &rarr;
+*Content* &rarr; *Categories*. To learn more about the process, go to the
+[Defining Categories for
+Content][/discover/portal/-/knowledge_base/7-1/organizing-content-with-tags-and-categories#defining-categories-for-content]
+section.
+
++$$$
+
+Note: Categories are designed to be nested into a hierarchy. While this allows
+multiple categories to be grouped together and handled as a single unit, it
+also makes it difficult for administrators to see the entire vocabulary. To
+view all categories at once, go to *Control Panel* &rarr; *Configuration*
+&rarr; *System Settings* &rarr; *Assets* and select *Flattened Tree* from the
+drop-down menu in *Asset Categories Web*. Then return to *Site Menu* &rarr;
+*Content* &rarr; *Categories*.
+
+$$$
+
+## Adding and Removing Products from Categories [](id=adding-and-removing-products-from-categories)
+
+To add a product to one or more categories:
+
+1.  Go to the catalog, select a product, and open its *Categorization* tab.
+
+2.  Click on the *Select* button under the label for the appropriate vocabulary,
+    and in the pop-up that appears the check the box next to each category that
+    applies.
+
+3.  Click Save.
+
+A product can be removed from a category the same way by unchecking the boxes in
+step 2. Alternatively, several products can be removed at once using the category
+application:
+
+1.  Find the category you want at *Site Menu* &rarr; *Content* &rarr;
+    *Categories*. Then click ![Options](../../images/icon-options.png)
+    &rarr; *Edit*, and select the *Products* tab.
+
+2.  Check the box next to each product you want to remove from the category.
+
+3.  Click *Remove*.
+
+## Other Category Associations [](id=other-category-associations)
+
+You may want to associate more than just products with your categories. Creating
+relationships with pages and images can make it easier to display and maintain
+category content.
+
+### Category Display Page [](id=category-display-page)
+
+The category display page is a feature that creates a relationship between
+a category, a specified page, and a URL for that page. This relationship makes
+it easy to display and organize content or applications related to to that
+category, as changes made to the category will be automatically reflected on the
```

Comment:
Don't use future tense. Change to "...changes made to the category are reflected automatically..."